target/linux/ramips/patches-5.4/0048-asoc-add-mt7620-support.patch

```diff
@@ -20,22 +20,22 @@ Signed-off-by: John Crispin <blogic@openwrt.org>
 
 --- a/arch/mips/ralink/of.c
 +++ b/arch/mips/ralink/of.c
-@@ -13,6 +13,7 @@
+@@ -14,6 +14,7 @@
  #include <linux/of_fdt.h>
  #include <linux/kernel.h>
  #include <linux/memblock.h>
 +#include <linux/module.h>
  #include <linux/of_platform.h>
  #include <linux/of_address.h>
  
-@@ -24,6 +25,7 @@
+@@ -25,6 +26,7 @@
  #include "common.h"
  
  __iomem void *rt_sysc_membase;
 +EXPORT_SYMBOL(rt_sysc_membase);
```

Comment:
This export was backported and the include is probably also not needed nay more.

Reply:
Is your recommendation to simply delete the include and rebase?

Reply:
I removed this part myself before pushing.